.github/workflows/test.yml

```diff
@@ -0,0 +1,68 @@
+name: Run API tests
```

Comment:
See comment about regarding the name.

Reply:
I've changed this one to TDR. There's a new build job as well which I've named with TDR as the prefix.